@uportal/esco-content-menu-lit/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@uportal/esco-content-menu-lit",
+  "version": "2.0.0",
+  "description": "Web component to display into eyebrow menu a user information menu with profile picture and more.",
+  "license": "Apache-2.0",
+  "author": "Julien Gribonvald <julien.gribonvald@gmail.com>",
+  "contributors": [
+    "Grégory Brousse <pro@gregory-brousse.fr>"
+  ],
+  "keywords": [
+    "apereo",
+    "uportal",
+    "content",
+    "menu",
+    "user-info"
+  ],
+  "main": "dist/js/esco-content-menu.js",
+  "dependencies": {
+    "@babel/runtime": "^7.18.6",
+    "@fortawesome/fontawesome-svg-core": "^6.1.1",
+    "@fortawesome/free-brands-svg-icons": "^6.1.1",
+    "@fortawesome/free-regular-svg-icons": "^6.1.1",
+    "@fortawesome/free-solid-svg-icons": "^6.1.1",
+    "@lit/localize": "^0.11.2",
+    "@uportal/open-id-connect": "^1.37.0",
+    "concurrently": "^7.2.1",
+    "core-js": "^3.22.8",
+    "lit": "^2.2.5",
+    "lodash": "^4.17.21",
+    "match-sorter": "^6.3.1",
+    "striptags": "^3.2.0",
+    "swiper": "^8.2.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.18.5",
+    "@babel/plugin-proposal-decorators": "^7.18.2",
+    "@babel/plugin-transform-runtime": "^7.18.6",
+    "@babel/preset-env": "^7.18.2",
+    "@babel/preset-typescript": "^7.17.12",
+    "@lit/localize-tools": "^0.6.3",
+    "@types/lodash": "^4.14.182",
+    "@typescript-eslint/eslint-plugin": "^5.27.1",
+    "@typescript-eslint/parser": "^5.27.1",
+    "acorn": "^8.7.1",
+    "babel-loader": "^8.2.5",
+    "babel-polyfill": "^6.26.0",
+    "css-loader": "^6.7.1",
+    "dotenv-webpack": "^7.1.0",
+    "eslint": "^8.17.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "html-webpack-plugin": "^5.5.0",
+    "json-server": "^0.17.0",
+    "prettier": "^2.6.2",
+    "sass": "^1.52.3",
+    "sass-loader": "^13.0.0",
+    "terser-webpack-plugin": "^5.3.3",
+    "ts-loader": "^9.3.0",
+    "typescript": "^4.7.3",
+    "webpack": "^5.73.0",
```

Comment:
I'm a bit cautious using webpack here.
Vue CLI is/was also webpack based, and even through it provided good defaults, it was still a pain to maintain.
The deep dependency tree frequently has vulns which cause noise as dependabot and renovate bot try to respond.
WebPack had (and may still have) issues around handling code in ESM format.
And when custom configuration is needed, it can be rather verbose and circuitous.

While not a blocker.
I'd lean strongly towards moving to another build system like:
* Rollup (https://lit.dev/docs/tools/production/#building-with-rollup)
* ESBuild (https://github.com/zandaqo/esbuild-plugin-lit#readme)
* Parcel (https://parceljs.org/)